### PR TITLE
Funding Page: Vested Token Fee amount should not be displayed as "Locked" in Claiming Box

### PR DESCRIPTION
**Pull request by vipergtsrz**
Added the "fee" logic to all the claiming calculations

### CODE REVIEW

File: src/funding/status-card/status-card.ts [hunk 1]
```diff
@@ -39,7 +39,13 @@ export class StatusCard {
   private isDaoFullyClaimed() : boolean {
     //return true if all tokens for this dao have been claimed
     if (!this.deal.isExecuted) return false;
-    return (this.tokenDao.tokens as ITokenCalculated[]).every(x => BigNumber.from(x.amount).eq(x.claimingClaimed?.add(x.instantTransferAmount) ?? 0));
+    return (this.tokenDao.tokens as ITokenCalculated[]).every(x => {
+      const totalAmount = BigNumber.from(x.amount);
+      const instantTransferAmount = BigNumber.from(x.instantTransferAmount);
+      const instantTransferAmountAfterFee = instantTransferAmount.sub(instantTransferAmount.mul(3).div(1000));
+      const swapFee = totalAmount.mul(3).div(1000);
```
Comment (dkent600) [inline src/funding/status-card/status-card.ts:46]: we really ought to have a single definition of the swap fee, not recomputed everywhere.  Could be a static value on DealService.

Comment (dkent600) [reply src/funding/status-card/status-card.ts:46]: and remember that bignumber does not do fractions

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/entities/DealTokenSwap.ts [hunk 2]
```diff
@@ -235,17 +237,25 @@ export class DealTokenSwap implements IDeal {
...
         if (!isClaimed) return; //immediately returns if it's already false from a previous loop
-        isClaimed = dao.tokens.every(daoToken => {
+        isClaimed = this.getOtherDao(dao).tokens.every(daoToken => {
           const tokenClaims = claims.filter(x => x.token.address === daoToken.address); //filter claims by token
           if (!tokenClaims) return false; //no claimed tokens exist for the given DAO and token
           const totalClaimed : BigNumber = tokenClaims.reduce((a, b) => a.add(b.claimedAmount), BigNumber.from(0));
-          return totalClaimed.add(BigNumber.from(daoToken.instantTransferAmount)).gte(daoToken.amount);
+          const instantTransferAmount = BigNumber.from(daoToken.instantTransferAmount);
+          const totalAmount = BigNumber.from(daoToken.amount);
+          const instantTransferAmountAfterFee = instantTransferAmount.sub(instantTransferAmount.mul(3).div(1000));
```
[the remark this answers is not in the document]
Comment (vipergtsrz) [reply src/entities/DealTokenSwap.ts]: Updated to use BigNumberJs service in one place

File: src/funding/status-card/status-card.ts (hunk 1 above)
Comment (vipergtsrz) [reply src/funding/status-card/status-card.ts]: Done

**Review by dkent600 — APPROVED** [review]

**Pull request merged**